src/rayon/map.rs

```diff
@@ -2,8 +2,8 @@
 //!
 //! You will rarely need to interact with this module directly unless you need to name one of the
 //! iterator types.
-//!
-//! Requires crate feature `"rayon"`
+
+#![cfg_attr(docsrs, doc(cfg(feature = "rayon")))]
```

Comment:
I think it would be better for both maintenance and coverage if we keep these next to the actual `#[cfg(feature)]` in question. In this case, that would be out in `lib.rs` on the entire `mod rayon`.